include/RAJA/policy/cuda/kernel/internal.hpp

```diff
@@ -191,7 +191,7 @@ struct CudaFixedMaxBlocksData
 };
 
 RAJA_INLINE
-int cuda_max_blocks(int block_size)
+size_t cuda_max_blocks(int block_size)
```

Comment:
block_size should not be size_t as it is num_threads

Reply:
The argument is of type int, do you mean the return type?